src/App/App.tsx

```diff
@@ -65,197 +65,195 @@ class App extends Component {
-                {/* New Focus Investigation Plan form view */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={NEW_PLAN_URL}
-                  component={NewPlan}
-                />
-                {/* Edit Focus Investigation Plan form view */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={`${PLAN_UPDATE_URL}/:id`}
-                  component={ConnectedUpdatePlan}
-                />
-                {/* Manage Plans list view */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={PLAN_LIST_URL}
-                  component={ConnectedPlanDefinitionList}
-                />
-                {/** Organization list view */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={ORGANIZATIONS_LIST_URL}
-                  component={ConnectedOrgsListView}
-                />
-                {/** organization create view */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={CREATE_ORGANIZATION_URL}
-                  component={ConnectedCreateEditOrgView}
-                />
-                {/** Organization edit view */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={`${EDIT_ORGANIZATION_URL}/:id`}
-                  component={ConnectedCreateEditOrgView}
-                />
-                {/* tslint:disable jsx-no-lambda */}
-                <Route
-                  exact={true}
-                  path={LOGIN_URL}
-                  render={routeProps => <OauthLogin providers={providers} {...routeProps} />}
-                />
-                <Route
-                  exact={true}
-                  path="/oauth/callback/:id"
-                  render={routeProps => (
-                    <ConnectedOauthCallback
-                      LoadingComponent={Loading}
-                      providers={providers}
-                      oAuthUserInfoGetter={oAuthUserInfoGetter}
-                      {...routeProps}
-                    />
-                  )}
-                />
-                {/* tslint:enable jsx-no-lambda */}
-                <ConnectedPrivateRoute
-                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                  exact={true}
-                  path={LOGOUT_URL}
-                  component={ConnectedLogout}
-                />
-              </Switch>
-            </Col>
-          </Row>
-        </Container>
```

Comment:
and this line was removed, everything else is just indentation.

Reply:
https://github.com/onaio/reveal-frontend/pull/502#issuecomment-539452200

Oh i see, it's to match the header's indentation. Sawa then just changing this to fluid makes sense.